js/libs/form.js

```diff
@@ -757,6 +757,19 @@ Fliplet.Widget.instance('form-builder', function(data) {
                   return { id: option };
                 });
 
+                if (field.value.length && field._type === 'flCheckbox') {
```

Comment:
let's nest the two blocks only when there's a value and use a switch case:

```
if (field.value) {
  switch (field._type) {
    case 'flCheckbox'
  }
}
```

Reply:
@squallstar fixed